web/src/features/panels/zone/FeedbackCard.tsx

```diff
@@ -0,0 +1,270 @@
+      text={buttonText}
+      isButton={true}
+      onClick={handleSave}
+    />
+  );
+}
+
+function FeedbackActions({
+  state,
+  setState,
+  estimationMethod,
+}: {
+  state: string;
+  setState: Dispatch<SetStateAction<string>>;
+  estimationMethod?: string;
+}) {
+  const [inputText, setInputText] = useState('');
+  const [feedbackScore, setfeedbackScore] = useState('');
+
+  const question = getQuestionTranslation('rate-question');
+
+  const handleInputChange = (event: { target: { value: SetStateAction<string> } }) => {
+    setInputText(event.target.value);
+  };
+
+  const handleSave = () => {
+    setState('3');
+    fetch(`https://hooks.zapier.com/hooks/catch/14671709/3l9daod/`, {
+      method: 'POST',
+      body: JSON.stringify({
+        score: feedbackScore,
+        feedback: inputText,
+        reference: estimationMethod,
+      }),
+    });
+  };
+
+  if (state === '3') {
+    return null;
+  }
+
+  return (
+    <div className="flex flex-col">
+      <div data-test-id="feedback-question" className="text-sm">
+        {question}
+      </div>
+      <ActionPills setState={setState} setFeedbackScore={setfeedbackScore} />
+      {state === '2' && (
+        <div>
+          <div className="my-3 h-[1px] w-full bg-neutral-200 dark:bg-gray-700" />
+          <div>
+            <InputField inputText={inputText} handleInputChange={handleInputChange} />
+            <SubmitButton handleSave={handleSave} />
+          </div>
+        </div>
+      )}
+    </div>
+  );
+}
+
+function ActionPills({
+  setState,
+  setFeedbackScore,
+}: {
+  setState: Dispatch<SetStateAction<string>>;
+  setFeedbackScore: Dispatch<SetStateAction<string>>;
+}) {
+  const agreeText = getQuestionTranslation('agree');
+  const [pillContent] = useState([1, 2, 3, 4, 5]);
+  const disagreeText = getQuestionTranslation('disagree');
+  const [CurrentPillNumber, setPillNumber] = useState('');
+
+  const handlePillClick = (identifier: string) => {
+    setFeedbackScore(identifier);
+    setPillNumber(identifier);
+    setState('2');
+  };
+
+  return (
+    <div className="flex w-full flex-col pt-2">
+      <PillContent
+        pillContent={pillContent}
+        handlePillClick={handlePillClick}
+        CurrentPillNumber={CurrentPillNumber}
+      />
+      <div className="flex flex-row items-center justify-between pt-1">
+        <div
+          data-test-id="disagree-text"
+          className="text-xs font-medium text-neutral-400"
+        >
+          {disagreeText}
+        </div>
+        <div data-test-id="agree-text" className="text-xs font-medium text-neutral-400">
+          {agreeText}
+        </div>
+      </div>
+    </div>
+  );
+}
+
+function PillContent({
+  pillContent,
+  handlePillClick,
+  CurrentPillNumber,
+}: {
+  pillContent: number[];
```

Comment:
If this is a string array we won't have to cast it to string below, but if possible it might be nicer to have them all as numbers and not cast them to strings if that works.

Reply:
It wasn't possible to use numbers all the way as the value of the `ToggleGroupPrimitive.Item` expected a string, so I changed it to a string array